projects/plugins/jetpack/extensions/blocks/subscriptions/menu.js

```diff
@@ -23,40 +24,58 @@ const NewsletterMenu = () => {
 	const accessLevel = useAccessLevel( postType );
 	const isPublished = postStatus === 'publish';
 
+	const { isUserConnected } = useConnection();
+	const connectUrl = `${ window?.Jetpack_Editor_Initial_State?.adminUrl }admin.php?page=my-jetpack#/connection`;
```

Comment:
Is there no helper for anything like this? 🤔 

Adding `?? ./` after the `adminUrl` might harden this more in case something in the chain doesn't exist.

Reply:
I asked @oskosk for a standard way, I am waiting for his reply. I think we can merge and iterate later :)

Reply:
@jeherve :wave: feels like appropriate to show above example to you too :-)

Reply:
That's interesting, I would have expected the wp-admin URL to be available directly from the Core Data store for example, but I can't find anything for it. Maybe that's something that should be added to Core?

If that's not possible, we could certainly extract an helper for `Jetpack_Editor_Initial_State.adminUrl` on our side, maybe in `@automattic/jetpack-shared-extension-utils`?

Reply:
For reference, I didn’t come up with it — I copied it from these components: https://github.com/search?q=repo%3AAutomattic%2Fjetpack+%24%7B+window%3F.Jetpack_Editor_Initial_State%3F.adminUrl+%7Dadmin.php%3Fpage%3Dmy-jetpack%23%2Fconnection%60&type=code